### PR TITLE
Spaces list page

### PR DESCRIPTION
**Pull request by mmmavis**
Relates to https://github.com/mozilla/mozfest-schedule-app/issues/18

I haven't built the "See all events in this Space" functionality yet.  I'll submit another PR after this gets merged.

mockup for reference:

mozfest 2016 mobile scheduler-05

### CODE REVIEW

File: js/schedule.js [hunk 1]
```diff
@@ -66,13 +70,17 @@ function Schedule(options) {
...
+                break;
+            case "_space":
+                break;
         }
     }
 
     // call makeSchedule() to display the selected list of sessions
     schedule.makeSchedule = function() {
         schedule.loadChosenTab();
-        schedule.$toggles.show();
```
Comment (mmmavis) [inline js/schedule.js]: removed this as we are not hiding the controls by default anymore

File: js/schedule.js [hunk 2]
```diff
@@ -429,8 +466,35 @@ function Schedule(options) {
         });
     }
     
+    // display the list of Spaces and their descriptions
+    schedule.displaySpacesList = function() {
+        schedule.clearHighlightedPage();
+        schedule.$pageLinks.find('#spaces-page-link').addClass('active');
+
+        schedule.loadSpaces(function() {
```
Comment (mmmavis) [inline js/schedule.js]: I can break this into smaller methods of `schedule` if necessary.

File: spaces.json [hunk 3]
```diff
@@ -0,0 +1,67 @@
+[
```
Comment (mmmavis) [inline spaces.json]: I grabbed content from https://2015.mozillafestival.org/sessions

**Comment by mmmavis** [comment]
@ryanpitts I would like to merge this into `gh-pages` but thought it'd be great to have this quickly checked by you first?

**Comment by ryanpitts** [comment]
LGTM :+1:

**Comment by mmmavis** [comment]
yay merging it nowwwww :tada:

**Pull request merged**